x-pack/plugins/dashboard_mode/public/index.scss

```diff
@@ -1,3 +0,0 @@
-@import 'ui/public/styles/styling_constants';
-
-@import 'core_plugins/kibana/public/dashboard/index';
```

Comment:
This was a workaround to get Kibana app styles into the dashboard app - no longer necessary.